src/validations/approveRequestValidation.js

```diff
@@ -0,0 +1,11 @@
+import Joi from 'joi';
+
+const approveRequestSchema = Joi.object().keys({
+  status: Joi.string()
+    .valid('approved', 'rejected', 'pending')
+    .required()
+    .min(4)
+    .label('Please Enter Correct Details')
```

Comment:
This is a label for this key, not the message that will be displayed when the validation fails. You can define custom messages through messages property.